pytorch_lightning/__init__.py

```diff
@@ -16,9 +16,10 @@
 from pytorch_lightning.callbacks import Callback  # noqa: E402
 from pytorch_lightning.core import LightningDataModule, LightningModule  # noqa: E402
 from pytorch_lightning.trainer import Trainer  # noqa: E402
+from pytorch_lightning.compressor.intel.neural_compressor import IncQuantization, QuantizationMode  # noqa: E402
 from pytorch_lightning.utilities.seed import seed_everything  # noqa: E402
 
-__all__ = ["Trainer", "LightningDataModule", "LightningModule", "Callback", "seed_everything"]
+__all__ = ["Trainer", "LightningDataModule", "LightningModule", "Callback", "seed_everything", "IncQuantization", "QuantizationMode"]
```

Comment:
```suggestion
__all__ = ["Trainer", "LightningDataModule", "LightningModule", "Callback", "seed_everything"]
```

Reply:
Only core Lightning objects should be exposed.

Reply:
removed